README.md

```diff
@@ -22,7 +22,7 @@ To compile you need the following tools:
 #### Building
 
 ```sh
-git clone --recursive https://github.com/jorisv/eigen-quadprog
+git clone --recursive https://github.com/jrl-umi3218/eigen-gurobi
```

Comment:
It seems like you based this PR on the wrong commit since this was in yesterday's PR.